.github/workflows/gradle.yaml

```diff
@@ -3,6 +3,12 @@ on:
 name: gradle build
 jobs:
   ubuntu:
+    services:
+      emulator:
+        image: gcr.io/cloud-spanner-emulator/emulator:latest
+        ports:
+          - 9010:9010
+          - 9020:9020
```

Comment:
Do we still need this, now that the harness tests have been moved to a separate runner?